changelog/fix-9542-rename-payouts-instant-depost-modal-flow

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button
```

Comment:
```suggestion
Significance: patch
Type: dev
Comment: Part of a larger change - renaming deposit to payout; this PR fixes instant payout modal + button

```

Reply:
Changed here, thank you! - [80cae5a](https://github.com/Automattic/woocommerce-payments/pull/9606/commits/80cae5a7e928f559a235727b196a706286f25d37)